bldnrnmacpkgcmake.sh

```diff
@@ -44,6 +44,11 @@ for i in $args ; do
   pythons="${pythons}${i};"
 done
 
+XQUARTZ_CMAKE_INCLUDE=""
+if test "$CPU" = "arm64" ; then
```

Comment:
Don't have access to an M1 and didn't really look into the X11 handling so far, but would it work to use `HINTS` or `PATHS` (https://cmake.org/cmake/help/latest/command/find_package.html) , i.e: 
```cmake
if(NRN_ENABLE_INTERVIEWS)
  # x11 is not required on windows
  if(NOT NRN_WINDOWS_BUILD)
    if(ARM64)
        find_package(X11 HINTS /usr/X11R6   QUIET)
   else()
       find_package(X11 QUIET)
   endif()
```

Reply:
Great suggestion. I'll try it out.

Reply:
Sadly, I was unsuccessful with that strategy. Still linking against homebrew
```
% otool -L `which idraw`
/Applications/NEURON/bin/idraw:
	/opt/homebrew/opt/libx11/lib/libX11.6.dylib (compatibility version 11.0.0, current version 11.0.0)
	/opt/homebrew/opt/libxext/lib/libXext.6.dylib (compatibility version 11.0.0, current version 11.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1292.100.5)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 905.6.0)
```

Reply:
Maybe PATH preference still persists despite `PATHS` or `HINTS`.  One last idea would be to try something similar to `Random123` handling here: https://github.com/BlueBrain/CoreNeuron/blob/master/CMake/AddRandom123Submodule.cmake#L14

Reply:
@nrnhines  Found this: https://gitlab.kitware.com/cmake/cmake/-/issues/20878 
If `CMAKE_IGNORE_PATH` works then we would only need to update `bldnrnmacpkgcmake.sh` and avoid adding generalized extra handling in CMake.